Submodels/MTSOL_call.py

```
@@ -1,20 +1,613 @@
+        iter_count : int
+            Number of iterations performed up until failure of the solver.
+
+        Returns
+        -------
+        None
+        """
+
+        # TODO: Write crash recovering code. 
+        #  potentiall uses the number of iterations up to failure to check the flowfield and determine the cause of the crash.
+
+        pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)